frontend/src/services/getConfig.js

```diff
@@ -3,6 +3,9 @@ const deepmerge = require('deepmerge');
 const { getLocales } = require('./getLocales');
 
 function getFiles(dir, files = []) {
+  if (!fs.existsSync(dir)) {
```

Comment:
`fs.existsSync()` is the synchronous version of `fs.exists()`.
`fs.exists()` is now deprecated and recommand using `fs.stats()` of `fs.access()` instead.
`fs.stats()` doesn't seems to have a sync version, but `fs.access()` does : `fs.accessSync()`.

So it might be better to use `fs.accessSync()`.

- https://nodejs.org/api/fs.html#fsexistspath-callback
- https://nodejs.org/api/fs.html#fsaccesssyncpath-mode

Reply:
Thanks for your advice :+1:
I added a commit about it